src/themes/material/components/edit-chip-list/edit-chip-list.component.ts

```diff
@@ -0,0 +1,198 @@
+import {
+	Component,
+	Input,
+	OnInit,
+	ChangeDetectorRef,
+	ViewChildren,
+	QueryList
+} from '@angular/core';
+
+import {AppError} from 'ng2-qgrid/core/infrastructure';
+import {Command, CommandManager} from 'ng2-qgrid/core/command';
+import {Shortcut, ShortcutManager} from 'ng2-qgrid/core/shortcut';
+import {EditChipComponent} from './edit-chip/edit-chip.component';
+import {ChipState} from './edit-chip/edit-chip.component';
+
+interface ISelected {
+	index?: number;
+	value?: string;
+	action: Actions;
+}
+
+interface IChipItem {
+	value?: string | any;
+	state?: string;
+	qGridFocusEnabled?: boolean;
```

Comment:
avoid to use qGrid prefix

Reply:
the code has been refactored in commit [refactoring: qGridFocus... properties renamed](https://github.com/qgrid/ng2/pull/39/commits/46b594341d4c95229d6419f373228890b2adefa5).
The properties 'qGridFocus(Enabled/Disabled)' has been renamed to 'focusDirective(Enabled/Disabled)'.